client/my-sites/site-settings/site-icon-setting/style.scss

```diff
@@ -15,6 +15,10 @@
 	cursor: pointer;
 }
 
+.site-icon-setting__icon .site-icon {
+  border: 1px solid transparent;
+}
+
 .site-icon-setting__icon:hover .site-icon {
 	border-style: dashed;
 	border-color: $gray-dark;
```

Comment:
To resolve review issue, we might consider moving styles into an `is-blank` modifier:

```scss
&.is-blank {
	border: 1px dashed $gray-dark;
}
```